### PR TITLE
new oai-pmh xwalk mapping

### PR DESCRIPTION
**Pull request by KarmeleJisc**

### CODE REVIEW

File: PublicationsRouter/v2/oai-pmh/XWALK.md [hunk 1]
```diff
@@ -7,16 +7,20 @@ Note that given the simplicity of the OAI_DC format, only a limited subset of th
...
-| metadata.author | oai_dc:creator |
-| metadata.author.affiliation | oai_dc:contributor |
-| metadata.article.language | oai_dc:langauage |
 | metadata.publication_date | oai_dc:date |
-| metadata.license_ref.title | oai_dc:rights |
-| metadata.article.subject | oai_dc:subject |
-
-(May 2015: Note that this output will be reviewed shortly with a view to expanding the information available via OAI-PMH so it is comparable to that which is available via the [API](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/OutgoingNotification.md).)
+| metadata.license_ref | oai_dc:rights |
+| metadata.embargo | oai_dc:rights |
```
[the remark this answers is not in the document]
Comment (adam-rehin) [reply PublicationsRouter/v2/oai-pmh/XWALK.md]: Other than that, can you get Jake to check that this matches the code.

[the remark this answers is not in the document]
Comment (KarmeleJisc) [reply PublicationsRouter/v2/oai-pmh/XWALK.md]: I don't think it is a good idea to omit the prefix, I can add a blank space if it makes this easy to read

This documentation is based on the code changes, I made it.

[the remark this answers is not in the document]
Comment (adam-rehin) [reply PublicationsRouter/v2/oai-pmh/XWALK.md]: You made what? The documentation or the code changes?

[the remark this answers is not in the document]
Comment (KarmeleJisc) [reply PublicationsRouter/v2/oai-pmh/XWALK.md]: both, first the code changes and then updated the documentation based on the code changes

**Pull request merged**